msg.go

```diff
@@ -124,6 +124,7 @@ type Msg struct {
 	Answer   []RR       // Holds the RR(s) of the answer section.
 	Ns       []RR       // Holds the RR(s) of the authority section.
 	Extra    []RR       // Holds the RR(s) of the additional section.
+	RawMsg   []byte     // The original byte stream if this message was decoded from wire format
```

Comment:
This is a big change and increases Msg's size *a lot* (doubles?)

Reply:
Yes, it will for incoming messages.  Couple other possibilities:  only do it for TSIG() messages or have a boolean that controls whether it's done or not.  Any of those better?  But that still requires that this field at least be added (which by itself doesn't increase the size) but not used.  That or maybe use the Decorate*() functions and come up with a way to customize the Msg()?